deployment/articles-dxp/01-saml/06-configuring-saml.markdown

```diff
@@ -0,0 +1,131 @@
+---
+header-id: configuring-saml
+---
+
+# Configuring SAML
+
+There are some ways of configuring the SAML plugin outside the UI. This is done
+via OSGi configuration files and by uploading metadata XML to configure how
+connections are negotiated. 
+
+## OSGi Configuration Properties
+
+As noted in the previous tutorials, anything related to configuring SP
+connections must be done through the SAML Admin UI where configurations are
+saved to Liferay's database. SP connections can no longer be made via properties
+files as they were in versions prior to 3.1.0. 
+
+| **Note:** Don't use OSGi `.config` files or @product@'s System Settings Control
+| Panel application to configure SAML providers (IdP or SP). The System Settings
+| UI is auto-generated, and is for advanced administrators. It does not perform the
+| enhanced validation on the fields that the SAML Admin UI performs, so it could
+| allow administrators to create invalid configurations.
+
+This is a portal instance scoped configuration which can be managed via OSGi
+Configuration Admin. The affected properties are those in the
+`SAMLProviderConfiguration` metatype:
```

Comment:
@sez11a 

Change: Replace the list with the actual property (method) names from https://github.com/liferay/liferay-portal-ee/blob/master-private/modules/private/apps/saml/saml-api/src/main/java/com/liferay/saml/runtime/configuration/SamlProviderConfiguration.java

Reply:
Done. Thank you!